vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/VividusToAzureDevOpsApplication.java

```diff
@@ -17,38 +17,34 @@
 package org.vividus.azure.devops;
 
 import java.io.IOException;
-import java.util.Properties;
 
 import org.springframework.beans.BeansException;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.ApplicationContext;
-import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
 import org.springframework.context.annotation.ImportResource;
 import org.springframework.context.annotation.PropertySource;
 import org.vividus.azure.devops.configuration.AzureDevOpsExporterOptions;
 import org.vividus.azure.devops.exporter.AzureDevOpsExporter;
-import org.vividus.util.property.PropertyParser;
+import org.vividus.exporter.config.VividusExporterCommonConfiguration;
 
 @SpringBootApplication
+@Import(VividusExporterCommonConfiguration.class)
 @EnableConfigurationProperties(AzureDevOpsExporterOptions.class)
-@ImportResource(locations = { "org/vividus/azure/devops/spring.xml", "org/vividus/http/client/spring.xml" })
+@ImportResource(locations = { "org/vividus/azure/devops/spring.xml", "org/vividus/http/client/spring.xml",
+        "org/vividus/util/spring.xml"})
 @PropertySource({
     "org/vividus/http/client/defaults.properties",
     "org/vividus/util/defaults.properties"
 })
+@SuppressWarnings("checkstyle:hideutilityclassconstructor")
 public class VividusToAzureDevOpsApplication
 {
     public static void main(String [] args) throws BeansException, IOException
     {
         ApplicationContext context = SpringApplication.run(VividusToAzureDevOpsApplication.class, args);
         context.getBean(AzureDevOpsExporter.class).exportResults();
     }
-
-    @Bean("propertyParser")
-    public PropertyParser propertyParser(Properties properties)
-    {
-        return new PropertyParser(properties);
-    }
```

Comment:
since it's changed we need to remove duplicating properties from this module as well